_includes/v21.1/backups/backup-options.md

```diff
@@ -2,6 +2,6 @@
 -----------------------------------------------------------------+-------------------------+------------------------------
 `revision_history`<a name="with-revision-history"></a>           | N/A                     | Create a backup with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), which records every change made to the cluster within the garbage collection period leading up to and including the given timestamp.
```

Comment:
Also, since this option was new in v20.2 and the changes are minimal, could you update that version too? We don't normally update versions too far back, but this is a small case where the effort is probably worth it.